service/authenticator/auth.go

```diff
@@ -85,6 +85,18 @@ func (a *service) Login(_ context.Context, username, password string) (string, e
 // Validate checks if the user has the capability to execute the specific
 // actions on a resource
 func (a *service) Validate(_ context.Context, userID *string, queries []*models.ValidationPair) ([]*models.ValidationResult, error) {
+	// skip db for services
+	if strings.HasPrefix(*userID, servicePrincipal) {
+		results := make([]*models.ValidationResult, len(queries))
+		for i, query := range queries {
+			results[i] = &models.ValidationResult{
+				Query:  query,
+				Result: true,
+			}
+		}
+		return results, nil
+	}
```

Comment:
This gives all services `root` access to all data?

Reply:
No, actual check happens here: https://github.com/iryonetwork/wwm/blob/master/service/authenticator/auth.go#L146

Reply:
ok